package.nls.json

```diff
@@ -35,6 +36,7 @@
     "vscode-tye.tasks.tye-run.properties.watch.description": "Watches for code changes for all projects.",
 
     "vscode-tye.views.help.name": "Help and Feedback",
-    "vscode-tye.views.services.contents": "The extension requires you to run Tye in a command window. This extension will attempt to access `http://localhost:8000/api/v1/services` by default.",
+    "vscode-tye.views.services.contents.notInstalled": "A compatible version of Tye has not been found. You may need to install a more recent version.\n[Install Latest Tye](command:vscode-tye.commands.help.installTye)",
```

Comment:
Nit: I am not sure if there are any guidelines about using the 'title case' or 'sentence case' for the buttons, but I feel that the 'sentence case' might be better for it. i.e. "Install latest Tye".

Reply:
VS Code seems to use "title case" for buttons (at least in its various welcome views), so I'll leave as-is for now.